registrar/apps/api/v1/mixins.py

```diff
@@ -323,3 +325,11 @@ def validate_enrollment_data(self, enrollments):
                 raise ValidationError(
                     'expected request dicts to have string value for "status"'
                 )
+            if course_id and enrollment.get('course_staff') is not None:
+                if not isinstance(enrollment.get('course_staff'), bool):
```

Comment:
```suggestion
            course_staff_flag = enrollment.get('course_staff')
            if course_id and course_staff_flag is not None:
                if not isinstance(course_staff_flag, bool):
```
Style nit: avoid local code duplication